src/@ionic-native/plugins/clover-go/index.ts

```diff
@@ -0,0 +1,90 @@
+import { Injectable } from '@angular/core';
+import { Plugin, Cordova, CordovaProperty, CordovaInstance, InstanceProperty, IonicNativePlugin } from '@ionic-native/core';
+import { Observable } from 'rxjs';
+
+/**
+ * @name Clover Go
+ * @description
+ * This plugin connect to Clover Go payment device and process payments.
+ *
+ * @usage
+ * ```typescript
+ * import { CloverGo } from '@ionic-native/clover-go';
+ *
+ *
+ * constructor(private cloverGo: CloverGo) { }
+ *
+ * ...
+ *
+ *
+ * this.cloverGo.init(configuration)
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * this.cloverGo.connect()
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * this.cloverGo.disconnect()
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * this.cloverGo.sale(saleInfo)
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'CloverGo',
+  plugin: 'cordova-plugin-clovergo', // npm package name, example: cordova-plugin-camera
+  pluginRef: 'clovergo', // the variable reference to call the plugin, example: navigator.geolocation
+  repo: 'https://github.com/hotwax/cordova-plugin-clovergo', // the github repository URL for the plugin
+  install: 'ionic plugin add cordova-plugin-clovergo', // OPTIONAL install command, in case the plugin requires variables
+  installVariables: [], // OPTIONAL the plugin requires variables
+  platforms: ['Android', 'iOS'] // Array of platforms supported, example: ['Android', 'iOS']
```

Comment:
Remove comments and empty `installVariables`property

Reply:
Removed